src/index.js

```diff
@@ -0,0 +1,25 @@
+import './style.css';
+
+const mainContainer = document.querySelector('.container');
+console.log(mainContainer);
+const todoTasks = [{
+  description: 'wash the dishes',
+  completed: true,
+  index: 1,
+},
+{
+  description: 'complete to do list project',
+  completed: true,
+  index: 2,
+},
+];
+
+document.addEventListener('DOMContentLoaded', () => {
+  todoTasks.forEach((n) => {
+    const output = document.createElement('div');
+    output.classList.add('listitem');
+    output.setAttribute('id', `${n.index}`);
+    output.innerHTML = `<div><input type="checkbox"> ${n.description}</div> <span class="listitemIcon"><i class="fa-solid fa-ellipsis-vertical"></i></span>`;
+  //  mainContainer.appendChild(output);
+  });
```

Comment:
- [ ] As I can see after running the project I can't see any tasks displayed so kindly fix the error so I can see some of the tasks as required so kindly fix it and submit it again 💯💪🏻💥

Reply:
Sure , Now it works fine 
@Reem-lab can you check again?